### DISCUSSION TITLE
CyclOSM / waymarkedtrails /

### DISCUSSION THREAD

**Discussion by manouche** [organicmaps/organicmaps#6348] [Ideas]
First I really love your app and the way it look :) I'ld love using it for my bike trip in forreign country with no 4G operators.

I found you with this good big comparaison-sheet but in french https://docs.google.com/spreadsheets/d/1w5sxpRLtmqXvC2lnHHc_XKeb0mM4uLpvVj10oq1xfOE/edit#gid=0

Only one tool is really missing for me, is the possibility to visualize "safe bike road" or "marked trail" like you do with metro. They are all shown on this page : https://cycling.waymarkedtrails.org/#?map=7.0/46.7339/8.1165
Are you planning to do so ?

There is also a very good cyclosm style existing in cartocss format, is it possible to convert it to your map style ?

**Comment by biodranik** [comment]
Yes, we plan a separate cycling style where a layer with routes from osm can be added. #2360
